src/Form/AuthenticationForm.php

```diff
@@ -565,10 +565,10 @@ protected function createDebugText(\Exception $exception, KeyInterface $key): st
   public function ajaxCallback(array &$form, FormStateInterface $form_state): AjaxResponse {
     $response = new AjaxResponse();
     // Get any status messages so they can be rendered in the placeholder.
-    $status_messages = $this->renderer->render(StatusMessages::renderMessages());
+    $form['messages']['#value'] = $this->renderer->render(StatusMessages::renderMessages());
```

Comment:
Still... Only variables should be passed by reference. 
https://github.com/drupal/core/blob/17557c97a816bc78e70989a86064c562bc27027f/lib/Drupal/Core/Render/RendererInterface.php#L335